docs/configuration.md

```diff
@@ -223,6 +237,31 @@ log:
     </td>
 </table>
 
+### hooks
```

Comment:
@moxiegirl This should be:

```
The `hooks` subsection configures the logging hooks' behavior.
```

The stuff about "disabled" and the loglevel section seems irrelevant. It should only be required to have `disabled` when you want to disable a hook (similar to how webhooks are implemented).

TODOs are okay rather than holding up good contributions on missing documentation.